src/action.js

```diff
@@ -46,6 +67,26 @@ async function getRepoOrigin() {
   return origin;
 }
 
+async function runPreflight() {
```

Comment:
I'm personally not a fan of "do a check and return some data" type of functions. SRP and all. We should have a "getEventData()" and a "runPreflight()" and pass the event data info the preflight check

Reply:
Ok. Will adjust.

Reply:
This is done. Resolving.